Problem building with Piranha

Hi! I'm trying to build `symengine.py` with `symengine` built against `piranha` on `OS X 10.11 El Capitan`. Although `symengine` itself builds fine, I hit this issue when building `symengine.py` (both `v0.2.0` and the git master branch):
```
In file included from /Users/jp/Programs/GitHub/Spack/spack/opt/spack/darwin-elcapitan-x86_64/clang-7.3.0-apple/symengine-develop-t5qygkkgym536nmvw6wnblly6pdyi55p/include/symengine/mp_wrapper.h:4:
/Users/jp/Programs/GitHub/Spack/spack/opt/spack/darwin-elcapitan-x86_64/clang-7.3.0-apple/symengine-develop-t5qygkkgym536nmvw6wnblly6pdyi55p/include/symengine/symengine_rcp.h:378:20: error: no matching constructor for initialization of 'const SymEngine::Integer'
    return rcp(new T(std::forward<Args>(args)...));
                   ^ ~~~~~~~~~~~~~~~~~~~~~~~~
/private/var/folders/lc/b4bjrtcn69d279m6rf4f42y00000gn/T/jp/spack-stage/spack-stage-bFCR0D/symengine.py-0.2.0/symengine/lib/symengine_wrapper.cpp:20451:105: note: in instantiation of function template specialization 'SymEngine::make_rcp<const SymEngine::Integer, int &>' requested here
    __pyx_v_self->__pyx_base.__pyx_base.thisptr = ((SymEngine::RCP<SymEngine::Basic const > )SymEngine::make_rcp<const SymEngine::Integer>(__pyx_v_i_));
                                                                                                        ^
/Users/jp/Programs/GitHub/Spack/spack/opt/spack/darwin-elcapitan-x86_64/clang-7.3.0-apple/symengine-develop-t5qygkkgym536nmvw6wnblly6pdyi55p/include/symengine/integer.h:17:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'int' to 'const SymEngine::Integer' for 1st argument
class Integer : public Number
      ^
/Users/jp/Programs/GitHub/Spack/spack/opt/spack/darwin-elcapitan-x86_64/clang-7.3.0-apple/symengine-develop-t5qygkkgym536nmvw6wnblly6pdyi55p/include/symengine/integer.h:208:17: note: candidate constructor not viable: no known conversion from 'int' to 'integer_class' (aka 'mp_integer<>') for 1st argument
inline Integer::Integer(integer_class i_) : i{std::move(i_)}
```

[Build log](https://github.com/symengine/symengine.py/files/505794/spack-build.out.txt) attached. Note: I've had success in building `symengine.py` without `piranha` support activated in `symengine`.

--- a/symengine/lib/symengine.pxd
+++ b/symengine/lib/symengine.pxd
@@ -21,7 +21,7 @@ cdef extern from 'symengine/mp_class.h' namespace "SymEngine":
         integer_class(int i)
         integer_class(integer_class)
         integer_class(mpz_t)
-        integer_class(const string &s, int base) except +
+        integer_class(const string &s) except +
     mpz_t get_mpz_t(integer_class &a)
     const mpz_t get_mpz_t(const integer_class &a)
     cdef cppclass rational_class:

--- a/symengine/lib/symengine_wrapper.pyx
+++ b/symengine/lib/symengine_wrapper.pyx
@@ -741,12 +741,12 @@ cdef class Integer(Number):
             # Too big, need to use mpz
             int_ok = False
             tmp = str(i).encode("utf-8")
-            i__ = symengine.integer_class(tmp, 10)
+            i__ = symengine.integer_class(tmp)
         # Note: all other exceptions are left intact
         if int_ok:
-            self.thisptr = symengine.make_rcp_Integer(i_)
+            self.thisptr = <RCP[const symengine.Basic]>symengine.integer(i_)
         else:
-            self.thisptr = symengine.make_rcp_Integer(i__)
+            self.thisptr = <RCP[const symengine.Basic]>symengine.integer(i__)
 
     def __hash__(self):
         return deref(self.thisptr).hash()

